frame/support/test/tests/pallet_ui/call_argument_invalid_bound.stderr

```diff
@@ -1,26 +1,26 @@
 error[E0277]: `<T as pallet::Config>::Bar` doesn't implement `std::fmt::Debug`
-  --> $DIR/call_argument_invalid_bound.rs:20:41
+  --> $DIR/call_argument_invalid_bound.rs:20:36
    |
 20 |         pub fn foo(origin: OriginFor<T>, bar: T::Bar) -> DispatchResultWithPostInfo {
-   |                                               ^ `<T as pallet::Config>::Bar` cannot be formatted using `{:?}` because it doesn't implement `std::fmt::Debug`
+   |                                          ^^^ `<T as pallet::Config>::Bar` cannot be formatted using `{:?}` because it doesn't implement `std::fmt::Debug`
```

Comment:
Possibly this span has moved here and in the other `call_argument_invalid_*` ui tests because of the change from tuple to struct variants for the pallet call enum?

Reply:
I think error message is still fine